app/import.go

```diff
@@ -93,18 +105,25 @@ func (a *App) BulkImport(fileReader io.Reader, dryRun bool, workers int) (*model
 		}
 
 		select {
-		case linesChan <- LineImportWorkerData{line, lineNumber}:
+		case linesChan <- LineImportWorkerData{lineData, lineNumber}:
 		case err := <-errorsChan:
 			if stopOnError(err) {
 				close(linesChan)
 				wg.Wait()
 				return err.Error, err.LineNumber
 			}
 		}
+
+		if err == io.EOF {
```

Comment:
Can you add a comment here just to make it easier for the casual reader to understand the reason for this check being in this particular place?

Reply:
I added a comment above for the give reason for this check.